server/DBConfig.js

```diff
@@ -0,0 +1,11 @@
+const { Pool } = require("pg");
+
+const videosPool = new Pool({
+  connectionString:
```

Comment:
You probably didn't mean to check this in either - try to have this use your `.env` file rather than putting a copy of the connection string here.